challenges/MIT-REA/README.md

```diff
@@ -0,0 +1,42 @@
+# ![MIT ReACT](https://raw.githubusercontent.com/code-for-venezuela/2019-april-codeathon/master/assets/mit_react.png) Where are the Venezuelans that have been displaced? 
+
+## TL;DR
+
+Help us create a comprehensive census of Venezuelans that could benefit from MIT ReACT program.
+
+## Backgroung 
+
+### About the organization
+
+The [MIT ReACT](https://react.mit.edu/) Certificate Program is part of the MIT ReACT Hub, an initiative that was launched in May 2017 when the challenge of refugee education was presented at MIT SOLVE. Professor [Admir Masic](https://cee.mit.edu/people_individual/admir-masic/) and now Vice-Chancellor Ian Waitz championed the creation of a center at MIT to design and deploy new learning opportunities for displaced populations around the world. Driven by Prof. Masic own experience as a refugee, ReACT has been offering new learning programs for displaced people where they live. By blending in-person workshops, online courses, and paid professional internships they currently offer career and academic pathways to students in 10 countries. 
+
+### Problem statement
+
+MIT ReACT provides a certificate in Computer and Data Science for displaced populations and refugees. The first class of students is progressing through the online learning portion of the program, mastering MIT edX courses in Computer Science and Data Analytics. The program has already seen important results among its participants:
+
+**95% Graduation Rate:**  One year after the start of the program, all students have passed the 5-course curriculum. 
+**50% Internship full-time job conversion:** ReACT partner organizations, converted half of the participants into to full time jobs.
+
+MIT ReACT and Code For Venezuela are partnering to expand this program and take it to Venezuelans. There are over [three million](https://www.unhcr.org/news/press/2018/11/5be4192b4/number-refugees-migrants-venezuela-reaches-3-million.html) forcibly displaced Venezuelans around the world who have been forced to leave their homes  and seek asylum or resettlement in a new country. 
+
+However, MIT ReACT and Code for Venezuela have a big challenge. **It is hard to track the location of displaced Venezuelans to  engage them and promote this program.**
+
+## Challenge
+
+As an initial phase for this project, we would like a report that enables us to build the network of potential candidates and help us do the sourcing of Venezuelan talent in tech. With this information, we will be able to connect them with the ReACT program. 
+
+The solution should provide answers to the following questions:
+* Where are all the displaced and migrants from Venezuela? 
```

Comment:
what do we mean here by displaced? Is everyone that left Venezuela considered displaced?

Reply:
Good call. I think we mean this: https://www.who.int/environmental_health_emergencies/displaced_people/en/

Going to add link to that.